src/App.js

```diff
@@ -50,15 +50,11 @@ function App() {
             element={
               <Students />
             } />
-        </Route>
-        <Route
-          path='/student/:id'
-          element={
-            <ProtectedRoute isAuth={isLoggedIn}>
```

Comment:
why remove this?

Reply:
I added it to the nested route so that the navbar can show

Reply:
so do you need this route at all here?